docs/arch/readme.md

```diff
@@ -0,0 +1,37 @@
+# Architecture Views
```

Comment:
Looks like all of the arch docs and diagrams from `jkeane/archdocs` got included here

Reply:
I pruned them from branch